### ISSUE TITLE
Find in Project - click on result fails to highlight line, if file was not open yet

### ISSUE THREAD

**Issue by eburi** [atom/atom#8904]
Atom: 1.0.15, OS: OSX 10.10.5, Added Packages: atom-beautify, atom-pair, blame, color-picker, editorconfig, git-log, jshint, merge-conflicts, pretty-json, tabs-to-space

If I search using "Find in Project", I get a tab "Project Find Results" listing the results it found.

When I click on a result, the corresponding file is opened. BUT only if the file was already open, the cursor jumps to the line with the search result. If that file was not open, the file is opened BUT the cursor is always found on the last line in the file.

Expected Behaviour: When clicking on a result, the file should be opened - if not open, yet - and the cursor should be placed on the line where the result is that I clicked on.

**Comment by jmtuley** [comment]
This happens for me as well, in OSX 10.10.5, Atom 1.0.19.

**Comment by eburi** [comment]
I found the problem be something related to the editorconfig-package. I deleted the folder ~/.atom/ and re-installed Atom. At this point Find in Files works as expected.
(Mac OSX El Capitan, Atom 1.0.19)


From there I started adding the packages that I had before and Find in Files show the faulty behaviour after adding the editorconfig-package.

There might be other packages causing this issue, too. But it's definitely not a vanilla Atom issue
